experimental/Schemes/BlowupMorphism.jl

```diff
@@ -475,3 +475,32 @@ function show_details(io::IO, Bl::BlowupMorphism)
   show_details(io,exceptional_divisor(Bl))
 end
 
+@attr AbsCoveredSchemeMorphism function isomorphism_on_complement_of_center(f::BlowupMorphism)
+  iso_dict = get_attribute(f, :isos_on_complement_of_center)
+  p = projection(f)
+  X = domain(f)
+  Y = codomain(f)
+  dom_cov = Covering([U for U in keys(iso_dict)])
+  inherit_glueings!(dom_cov, default_covering(X))
+  cod_cov = Covering([codomain(p) for p in values(iso_dict)])
+  inherit_glueings!(cod_cov, default_covering(Y))
+  XU = CoveredScheme(dom_cov)
+  YV = CoveredScheme(cod_cov)
+  p_res_cov = CoveringMorphism(dom_cov, cod_cov, iso_dict)
+  p_res = CoveredSchemeMorphism(XU, YV, p_res_cov)
+
+  # Assemble the inverse
+  iso_inv_dict = IdDict{AbsSpec, AbsSpecMor}()
+  for U in keys(iso_dict)
+    q = iso_dict[U]
```

Comment:
```suggestion
  for (U,q) in iso_dict
```

Reply:
Thanks for the hint! I didn't know that.